Sources/PointFree/Debug.swift

```diff
@@ -2,22 +2,23 @@ import Either
 import Prelude
 
 extension EitherIO {
-  public func debug<B>(prefix: String = "", _ inspect: @escaping (A) -> B) -> EitherIO {
+  public func debug<B>(prefix: @autoclosure @escaping () -> String = "", _ inspect: @escaping (A) -> B) -> EitherIO {
```

Comment:
Sneaking in a small, unrelated fix. This eagerness led to printing outdated info, like the database connection URL, which I noticed when troubleshooting a Heroku db.